client/src/components/pages/Home.js

```diff
@@ -36,6 +36,11 @@ class Home extends Component {
 			selectedVideo: videoItems[0]
 		});
 	};
+	handleSave = async video => {
+		const result = await API.saveVideo(video);
+
+		if (result) alert('Video is added to your Playlist');
```

Comment:
Result will always be truthy so the best indication that the video was saved is to check if result has 200 status code.